release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1319,7 +1319,7 @@ In {product-title} 4.11, support for VMware ESXi 7.0 Update 1 or earlier is depr
 [id="ocp-4-11-deprecated-features-crio-parameters"]
 ==== Support for `pidsLimit` and `logSizeMax` CRI-O parameters will be deprecated
 
-In {product-title} 4.11, support for the `pidsLimit` and `logSizeMax` CRI-O parameters is expected to be deprecated in a future release. It is recommended to use the `podPidsLimit` and the `ContainerLogMaxSize` fields in the `KubeletConfig` CR instead. The default value of the `podPidsLimit` field is `4096`.
+In {product-title} 4.11, the `pidsLimit` and `logSizeMax` the `ContainerRuntimeConfig` CR will be deprecated and removed in a future release. Use the `podPidsLimit` and the `ContainerLogMaxSize` fields in the `KubeletConfig` CR instead. The default value of the `podPidsLimit` field is `4096`.
```

Comment:
nit: 
```
`logSizeMax` fields in the `ContainerRuntimeConfig`...
```

Reply:
looks good, I am not sure if  additional **the** is required in the below:
 
Use the `podPidsLimit` and **the** `ContainerLogMaxSize` fields in the `KubeletConfig` CR